src/pages/configeditor/migrating_data.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Configuration Editor Migration"
+excerpt: "Configuration Editor Migration."
+categories: configeditor
+slug: migration
+toc: true
+---
+
+## Migrating Early Access Program Data
+
+Customers that were part of the Early Access Program (EAP) can follow these steps to migrate existing artifacts to the newly installed Merative ACD Configuration Editor instace.
+
+1.  Stop the Merative Configuration Editor from the `acd-ce` diretory.
+
+    ```
+    ./run-acd-ce.sh -stop
+    ```
+
+1.  Navigate to the %HOME%/installs/config/artrifactstore directory in the EAP enviroment.
+1.  Create an archive of the tenent data directories.
+
+    ```
+    tar -czv --exclude=SHAREDBASE <archive-name>.tar.gz *
+    ```
+
+1.  Copy the archive to the Merative Configuration Editor environment.
+1.  Navigate to the %HOME%/installs/config/artrifactstore directory in the Merative enviroment.
+1.  Extract the archive to the .../artifactstore directory.
+
+    ```
+    tar xvf <archive-dir>/<archive-file> .
+    ```
+
+1.  Start the Merative Configuration Editor from the 'acd-ce' directory.
```

Comment:
Use back ticks around `acd-ce` for consistency, like you have above.
Same comment as above.  Suggestion is to re-phrase the sentence to say:  From the `acd-ce` directory, start the Merative Configuration Editor.